### PR TITLE
feat: Add InformationBar and mobile filters

### LINKED ISSUE #230: Create the filtering mobile menu
In order to apply filters in the mobile section, create the mobile filtering modal that can be found in the Marketplace UI as the [AssetFiltersModal.tsx](https://github.com/decentraland/marketplace/blob/d1f57b0afa314c81710efc3d2bbdc737ef0023df/webapp/src/components/Modals/AssetFiltersModal/AssetFiltersModal.tsx). Take into consideration that a second filtering modal might be required when implementing the Assets section as the filters are diferente.
Use the pagination hook to change the filters in the component as well.

### LINKED ISSUE #231: Build the AssetsInformation component
In order to:
- Show the users the total amount of asset.
- Add the sorting filters
- Enable users to open the filters modal in the mobile version of the site.

Extract the code from the [AssetTopbar](https://github.com/decentraland/marketplace/blob/d463e0c35383fa26fa5a27809126ffd5c278764f/webapp/src/components/AssetTopbar/AssetTopbar.tsx) to build a component that does what was described before.
The component should not include the selection / de-selection of filters.

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBa1k4IiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--80f3df9b0aed6990eba1da584c58a051ccec4278/Screenshot%202023-08-24%20at%2015.56.31.png)

### PR DESCRIPTION
**Pull request by LautaroPetaccio**
Closes #231 
Closes #230

### CODE REVIEW

File: src/components/Modals/CreationsFiltersModal/CreationsFiltersModal.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  )
+  const handleChangeRarity = useCallback(
+    (rarities: string[]) => {
+      setFiltersToChange({ ...filtersToChange, rarities: rarities as Rarity[] })
+    },
+    [filtersToChange]
+  )
+  const handleChangeStatus = useCallback(
+    (value: AssetStatus) => {
+      console.log('Changing status')
```
Comment (meelrossi) [inline src/components/Modals/CreationsFiltersModal/CreationsFiltersModal.tsx:49]: :)

Comment (LautaroPetaccio) [reply src/components/Modals/CreationsFiltersModal/CreationsFiltersModal.tsx:49]: LOL, removed it!

File: src/components/Modals/CreationsFiltersModal/CreationsFiltersModal.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      setFiltersToChange({ ...filtersToChange, status: getItemSaleStatus(value) })
+    },
+    [setFiltersToChange]
+  )
+
+  const handleClearFilters = useCallback(() => {
+    setFiltersToChange({ category: undefined, rarities: undefined })
```
[the remark this answers is not in the document]
Comment (LautaroPetaccio) [reply src/components/Modals/CreationsFiltersModal/CreationsFiltersModal.tsx:56]: We do, great catch!

File: src/lib/pagination.ts [hunk 3]
```diff
@@ -85,6 +86,26 @@ export function usePagination<T extends string = string, T2 extends string = str
...
+  const changeFilters = useCallback(
+    (filters: Partial<Record<T, string | undefined>>, options: { clearOldFilters: boolean } = { clearOldFilters: false }) => {
+      const params = new URLSearchParams(options.clearOldFilters ? {} : search)
+      // Reset the page when changing the filter
+      params.set('page', '1')
+      if (options.clearOldFilters && sortBy) {
+        params.set('sortBy', sortBy)
+      }
+      Object.entries<string | undefined>(filters).forEach(([filter, value]) => {
+        if (value) {
```
[the remark this answers is not in the document]
Comment (LautaroPetaccio) [reply src/lib/pagination.ts:98]: You're completely right, changed it!

File: src/components/InformationBar/InformationBar.tsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        <div className={styles.countContainer} data-testid={INFORMATION_BAR_COUNTER_DATA_TEST_ID}>
+          <p className={styles.countText}>{getCountText(count)}</p>
+        </div>
+      ) : null}
+      <div className={styles.rightOptionsContainer}>
+        <Dropdown direction="left" value={sortBy} options={sortByOptions} onChange={handleOnSortChange} />
+        {isMobile ? (
+          <i
+            data-testid={INFORMATION_BAR_MOBILE_DATA_TEST_ID}
+            className={classNames(styles.openFilters, styles.openFiltersWrapper, hasFiltersEnabled && styles.active)}
```
[the remark this answers is not in the document]
Comment (LautaroPetaccio) [reply src/components/InformationBar/InformationBar.tsx:29]: Done!

**Review by meelrossi — APPROVED** [review]

**Pull request merged**